main.py

```diff
@@ -42,6 +43,12 @@
 app.config['SECURITY_REGISTERABLE'] = False
 app.config['SECURITY_USER_IDENTITY_ATTRIBUTES'] = 'username'
 app.config['BABEL_DEFAULT_LOCALE'] = 'he'
+app.config['OAUTH_CREDENTIALS'] = {
```

Comment:
These should be environment variables that we'll set on the server.
You can't save a secret code in the code when it's open source.